snap/snapcraft.yaml

```diff
@@ -1,5 +1,5 @@
 name: openstack-hypervisor
-version: yoga
+version: antelope
```

Comment:
As we're now aligning to the upstream release numbering please can you update to 2023.1 for the version and then I think we're good to land.

Reply:
Done